.circleci/config.yml

```diff
@@ -670,31 +676,6 @@ jobs:
             other.test_memory64_proxies
             other.test_failing_growth_wasm64"
       - upload-test-results
-  test-wasm64-4gb:
-    environment:
-      LANG: "C.UTF-8"
-      # Only run 2 tests at a time to avoid OOM (since each tests used >4gb)
-      EMCC_CORES: "2"
-    # We don't use `bionic` here since its too old to run recent node versions:
-    # `/lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found`
-    executor: linux-python
-    steps:
-      - prepare-for-tests
-      # The linux-python image uses /home/circleci rather than /root and jsvu
-      # hardcodes /root into its launcher scripts so we need to reinstall v8.
-      - run: rm -rf $HOME/.jsvu
-      - install-v8
-      - install-node-canary
-      # When running wasm64 tests we need to make sure we use the testing
-      # version of node (node canary) when running the compiler output (e.g.
-      # in configure tests.
-      - run:
-          name: configure node canary
-          command: echo "NODE_JS = NODE_JS_TEST" >> ~/emsdk/.emscripten
-      - run-tests:
-          title: "wasm64_4gb"
-          test_targets: "wasm64_4gb"
-      - upload-test-results
   test-jsc:
```

Comment:
Oh, ignore the previous comments - I see you replaced the old one with the new.

How about just commenting out the old, and leaving the name of the remaining mode as 4gb? I do think/hope we can re-enable the old one. And it would be less confusing to avoid name changes over time.